packages/near-fast-auth-signer/src/components/AuthCallback/AuthCallback.tsx

```diff
@@ -40,11 +41,14 @@ const onCreateAccount = async ({
   email,
   gateway,
 }) => {
+  // Stop from LAK with multi-chain contract
+  const containBlacklistedContractId = getMultiChainContract() === contract_id;
```

Comment:
IMO we should throw earlier i.e. in the create account and add device pages to save the user the trouble. It should throw only if multichain contract is explicitly passed. If no value is passed we can continue the flow and only authenticate the signer